CONTRIBUTING.md is out of date

The Contributing guidelines indicate that the `make lint` linter [needs to be executed](https://github.com/influxdata/telegraf/blob/master/CONTRIBUTING.md#execute-linter) before submitting a PR, which was available in the Makefile [here](https://github.com/influxdata/telegraf/blob/bb14589469f9aaaadfd511cfec32354b387783d9/Makefile#L45-L46) in bb14589. However, it got changed in #3731, when Circle 2.0 CI was added and I'm guessing that it should be changed to `make check` now.

### System info:

Telegraf master 6bf56433

### Steps to reproduce:
```
> make lint
```

### Expected behavior:

Runs linter

### Actual behavior:
```
make: *** No rule to make target `lint'.  Stop.
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -477,7 +477,7 @@ the short tests.
 
 ### Execute linter
 
-execute `make lint`
+execute `make check`
 
 ### Execute short tests
 

